src/main/resources/com/cloudbees/jenkins/support/SupportAction/BundleBrowser.jelly

```diff
@@ -0,0 +1,57 @@
+<!--
+ ~ The MIT License
+ ~
+ ~ Copyright (c) 2013, CloudBees, Inc.
```

Comment:
This should probably be changed to 2016 and remove `CloudBees, Inc.` in the title.